src/integrations/bare.ts

```diff
@@ -45,7 +45,9 @@ export class ExpoBareIntegration {
     });
 
     DEFAULT_EXTRAS.forEach((extra) => {
-      if (Constants.hasOwnProperty(extra)) {
+      if (Object.hasOwnProperty.call(Device, extra)) {
```

Comment:
I think since there's only 2 default extras, it might be better to instead edit [L43-45](https://github.com/expo/sentry-expo/pull/220/files#diff-bf2a2e75e96b8f8baebacda5e0d026d854d776c311c22206d1f5e3851864b523R43-R45) to:

```
    setExtras({
      manifest,
      deviceYearClass: Device.deviceYearClass,
      linkingUri: Constants.linkingUri,
    });
 ```

Reply:
Go for it 😃 I won't have time to get back to this for some time